template/assets/use_case/template/submission/solution_q3as.ipynb.hbs

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting an API Key\n",
+    "\n",
+    "## {{title}} Submission\n",
+    "\n",
+    "To run your algorithms in the cloud, you have to create an API key and load it into your Credentials\n",
+    "\n",
+    "Start by visiting https://q3as.aqora.io and signing in with your GitHub or Google account. Click on your profile in the top right and go to API Keys. Tap Add API Key and enter a description for your API key. Tap Copy JSON to clipboard and paste the result in a file on your compute\n",
+    "\n",
+    "\n",
+    "> ⚠️ **Warning: Be Cautious with Secrets!**\n",
```

Comment:
hmm, this is a good point. unfortunately because the way that notebooks and the translation to scripts works, using things like `importlib.resources` is usually the best bet, but it requires that the file be in the package which means it would get uploaded. to handle this correctly, I would say we should make passing the credentials by environment variables possible which would require modifying the q3as client i believe.

Reply:
What would be really cool too if we use environment variables, is to automatically add q3as credentials to aqora with `aqora q3as add-key <PUBLIC> <SECRET>` and then have aqora inject it directly into the virtual environment paths when doing aqora install / test so the user just has to set it up once

Reply:
> hmm, this is a good point. unfortunately because the way that notebooks and the translation to scripts works, using things like `importlib.resources` is usually the best bet, but it requires that the file be in the package which means it would get uploaded. to handle this correctly, I would say we should make passing the credentials by environment variables possible which would require modifying the q3as client i believe.

But for now, the user doesn't send a file. Instead, I ask them to enter their credentials in the notebook. Shouldn't we handle it differently?



> What would be really cool too if we use environment variables, is to automatically add q3as credentials to aqora with `aqora q3as add-key <PUBLIC> <SECRET>` and then have aqora inject it directly into the virtual environment paths when doing aqora install / test so the user just has to set it up once

Maybe is another story if that work for you?